runtime/onert/core/src/compiler/ExecutorFactory.cc

```diff
@@ -221,7 +221,7 @@ void ExecutorFactory::prepareExternalTensors(ir::LoweredGraph &lowered_graph,
           // If an OpSequence input/output tensor does not have a own tensor object,
           // it must be using external tensors, so find the tensor from other tensor builders and
           // set the tensor to this tensor builder if portable
-          if (!backend_ctx->tensor_builder->tensorAt(ind))
+          if (!backend_ctx->tensor_builder->tensorRegistry()->getITensor(ind))
```

Comment:
Not a scope of this PR but
what if we store `tensorRegistry` into `backend_ctx` instead of getting it via `tensor_builder`, e.g., 

```
          if (!backend_ctx->tensorRegistry()->getITensor(ind))
```

Since `tensorRegistry` is a kinda global (?) registry whose lifetime spans from compilation to the end of execution. Therefore, I guess that it would be easier if we access `tensorRegistry` through one object (`backend_ctx`).

Reply:
Good point. It is going to be done, planned in #3809 already.